### PR TITLE
Hires Addition

### PR DESCRIPTION
**Pull request by hipsterusername**
Added Hires option to the Web UI and metadata

### CODE REVIEW

**Comment by hipsterusername** [comment]
@blessedcoolant ☝️

**Comment by hipsterusername** [comment]
Also @psychedelicious , since you probably have an opinion on options handling, and I'm the magicians apprentice, meddling with powers far beyond my skill level. ;)

**Comment by blessedcoolant** [comment]
Thanks for the PR. Code seems to be in order but I haven't tested it yet. I'll give it a shot in a bit. Anyone testing this PR, please also check the CLI. Because it makes changes to the generate code.

**Comment by blessedcoolant** [comment]
The implementation seems fine. Can't find any glaring issues. I'll wait for one more person to test this.

@lstein KSamplers with --hires_fix seem to be broken. They do not converge.

File: frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx [hunk 1]
```diff
@@ -217,6 +218,13 @@ const ImageMetadataViewer = memo(
                   onClick={() => dispatch(setWidth(seamless))}
                 />
               )}
+              {hires_fix && (
+                <MetadataItem
+                  label="High Resolution Optimization"
+                  value={hires_fix}
+                  onClick={() => dispatch(setWidth(hires_fix))}
```
Comment (psychedelicious) [inline frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx:225]: This should be `setHiresFix()`

**Review by psychedelicious — CHANGES_REQUESTED** [review]
I have only reviewed the code - not tested yet. Requested one change. I hope to test later this evening.

File: frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx (hunk 1 above)
Comment (hipsterusername) [reply frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx:225]: @psychedelicious thanks for the review!

That makes sense to me, but had mostly been using the “seamless” implement as reference. 

Is the seamless code (above it) also supposed to be setSeamless? Or how do these two differ

Thx - still figuring a few things out in the implementation :)

Comment (psychedelicious) [reply frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx:225]: You are correct, it should be setSeamless. These are the buttons to recall individual parameters

**Comment by hipsterusername** [comment]
Updated and fixed seamless along with it

**Comment by hipsterusername** [comment]
Updated tooltip to reflect changes, and snuck some other tooltip updates in as well

File: frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx (hunk 1 above)
Comment (psychedelicious) [reply frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx]: The calls are now broken in a different way hehe gotta provide the function with arguments

Comment (hipsterusername) [reply frontend/src/features/gallery/ImageMetaDataViewer/ImageMetadataViewer.tsx]: Getting there. ;) 

Updated to include (what I hope) are the correct arguments

**Comment by blessedcoolant** [comment]
I've tried to rebase this but sadly it wouldn't let me do it without a merge. This added commits but I'm not sure how it'll go if I merge.

@lstein Is this a clean merge or should we ask @hipsterusername to rebase this as a fresh PR against the current dev?

**Comment by hipsterusername** [comment]
I'm happy to do so later today and think its probably for the best - it shouldn't be too difficult.

**Comment by hipsterusername** [comment]
See #1063 for new version - same great hires flavor

**Pull request closed without merging**